doc-files/notes/24/JDK-8340852-info.md

```diff
@@ -0,0 +1,54 @@
+# Additional Information for JDK-8340852
```

Comment:
minor suggestion:
Rename the note to something like `JDK-8340852-ScrollPane.md`

Imagine a release with 10-20 such notes/directories, it might be helpful to add a bit of context to the name, don't you think?

Reply:
Renamed as suggested.